packages/next/src/build/webpack/loaders/next-serverless-loader/utils.ts

```diff
@@ -12,7 +12,7 @@ import type { BaseNextRequest } from '../../../../server/base-http'
 import type { __ApiPreviewProps } from '../../../../server/api-utils'
 
 import { format as formatUrl, UrlWithParsedQuery, parse as parseUrl } from 'url'
-import { parse as parseQs, ParsedUrlQuery } from 'querystring'
+import { ParsedUrlQuery } from 'querystring'
```

Comment:
Should we remove this type too?

Reply:
Seems ok to keep this type for now as it's referenced quite a bit, we can investigate having our own type to replace this with in a follow-up

Reply:
Can we change to “import type” so it’s clear?

Reply:
Updated to `import type`